### ISSUE TITLE
Monospace editor font not loading in Linux

### ISSUE THREAD

**Issue by tjblackheart** [a85/POSTMan-Chrome-Extension#7434]
**Describe the bug**
Using Postman on Linux, I have found that the font in the requester window is actually set fixed to "Droid Sans" (in `app/resources/app/js/0.js`). I don't have this one installed, so I get the default sans-serif for my system, which looks ... not so nice. 

**To Reproduce**
1. Remove Droid Sans from your system
2. Do a request
3. Look at the requester panel

**Expected behavior**
Postman should use the systems default monospace font.

**Screenshots**
![image](https://user-images.githubusercontent.com/5941480/66201509-ce7f0780-e6a3-11e9-8900-254080459c5b.png)

**App information (please complete the following information):**
 - App Type: native
 - Postman Version: 7.8.0
 - OS: Arch Linux

**Comment by kevin-postman** [comment]
@tjblackheart Can you tell me which version of Arch Linux you were using? I tried removing fonts on my Ubuntu 18.04, to reproduce this issue, and see that the font looks fine still. Postman ships with a monospace font in the case there isn't one available. I will grab the version of Arch Linux you have and load it up and try as well to see if I can reproduce it.

**Comment by tjblackheart** [comment]
Hi,

it's a standard arch install (which is a rolling release), so no special version here, just grab the latest ISO. My current desktop env is Cinnamon, and the default monospace font there is set to "Roboto Mono". The arch package for this is called "ttf-roboto-mono". No other desktops are installed.

I just tried again to remove "ttf-droid" and postman again uses the default serif font. Perhaps there is something different in the desktop configs in Cinnamon?

Anyway, thanks for looking!

**Comment by cmpickle** [comment]
For anyone else running into this problem I was able to resolve it by fixing my /etc/fonts/local.conf. The problem was the lines
```
<match>
     <edit mode="prepend" name="family"><string>Noto Sans</string></edit>
</match>
```
This makes it Noto Sans a legal match for all font types. After removing this line postman started to correctly use monospace fonts.

**Comment by kevin-postman** [comment]
Hi all. I am happy to report we have a fix for this in 7.19 release. Please give it a try and let us know if it solves the issue.
